src/main/java9/module-info.java

```diff
@@ -0,0 +1,9 @@
+module org.codehaus.plexus.interpolation {
+    exports org.codehaus.plexus.interpolation;
+    exports org.codehaus.plexus.interpolation.fixed;
+    exports org.codehaus.plexus.interpolation.multi;
+    exports org.codehaus.plexus.interpolation.object;
+    exports org.codehaus.plexus.interpolation.os;
+    exports org.codehaus.plexus.interpolation.reflection;
+    exports org.codehaus.plexus.interpolation.util;
```

Comment:
@rfscholte @slachiewicz do we need export all packages ....?

eg `org.codehaus.plexus.interpolation.os` look like internal one?

Reply:
my hope was only `org.codehaus.plexus.interpolation;`

however, a quick scan through the Maven codebase:
DefaultToolchainsBuilderTest uses org.codehaus.plexus.interpolation.os.OperatingSystemUtils
PathTranslatingPostProcessor uses org.codehaus.plexus.interpolation.util.ValueSourceUtils

as long as Maven is not using the modulepath that's not an issue.

I would probably go for only org.codehaus.plexus.interpolation and investigate if there are requests for opening more packages